Formula/a/apprise.rb

```diff
@@ -16,16 +16,24 @@ class Apprise < Formula
   end
 
   depends_on "python-setuptools" => :build
-  depends_on "python-click"
   depends_on "python-markdown"
   depends_on "python-requests-oauthlib"
   depends_on "python@3.12"
-  depends_on "pyyaml"
 
   def python3
     "python3.12"
   end
 
+  resource "click" do
+    url "https://files.pythonhosted.org/packages/96/d3/f04c7bfcf5c1862a2a5b845c6b2b360488cf47af55dfa79c98f6a6bf98b5/click-8.1.7.tar.gz"
+    sha256 "ca9853ad459e787e2192211578cc907e7594e294c7ccc834310722b41b9ca6de"
+  end
+
+  resource "pyyaml" do
+    url "https://files.pythonhosted.org/packages/cd/e5/af35f7ea75cf72f2cd079c95ee16797de7cd71f29ea7c68ae5ce7be1eda0/PyYAML-6.0.1.tar.gz"
+    sha256 "bfdf460b1736c775f2ba9f6a92bca30bc2095067b8a9d77876d1fad6cc3b4a43"
+  end
+
   def install
     system python3, "-m", "pip", "install", *std_pip_args, "."
```

Comment:
need to switch to a venv (which also requires an `include Language::Python::Virtualenv`):
```suggestion
    virtualenv_install_with_resources
```

Reply:
Good point, had totally forgotten this step